File: app/src/main/java/com/give3/gizrog/give3/HomeActivity.java
package com.give3.gizrog.give3;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class HomeActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_home);

        setupStartButton();
    }

    private void setupStartButton() {
        Button btn = findViewById(R.id.startButton);
        btn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent sectionsIntent = MenuActivity.makeIntent(HomeActivity.this);
                startActivity(sectionsIntent);
            }
        });
    }
}

File: app/src/main/java/com/give3/gizrog/give3/HomeActivity.kt
package com.give3.gizrog.give3

import android.content.Context
import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button

class HomeActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_home)

        setupStartButton()
    }

    private fun setupStartButton() {
        val btn = findViewById<Button>(R.id.button_start)
        btn.setOnClickListener {
            val sectionsIntent = MenuActivity.makeIntent(this@HomeActivity)
            startActivity(sectionsIntent)
        }
    }

    companion object {

        private val TAG = "MenuActivity"


        fun makeIntent(ctx: Context): Intent {
            return Intent(ctx, HomeActivity::class.java)
        }
    }
}
